src/components/Quotes.js

```diff
@@ -0,0 +1,49 @@
+import { useState, useEffect } from 'react';
+
+const Quotes = () => {
+  const [quote, setQuote] = useState(null);
+  const [isLoading, setIsLoading] = useState(true);
+  const [hasError, setHasError] = useState(false);
+
+  useEffect(() => () => {
+    const fetchData = async () => {
+      try {
+        const res = await fetch('https://api.api-ninjas.com/v1/quotes', {
+          headers: { 'X-Api-Key': 'CmTKM6B4Q5xkeSVvEc3FObE4vXWoMUq0yCAe0L0D' },
+        });
+        const quoteArray = await res.json();
+        const random = quoteArray[0];
+        setQuote(random);
+        setIsLoading(false);
+      } catch (error) {
+        setHasError(true);
+        setIsLoading(true);
+      }
+    };
+    fetchData();
+  }, []);
```

Comment:
- [ ] Nice work so far 👍🏿 it will be great if you add some dependence to your useEffect as what you have only runs once the page loades. But adding some dependence to the useEffect the quotes rerenders and gets new qoutes

Reply:
I understand that sometimes is uncomfortable receiving required changes. But it improves me.

Thanks Very much for the review and feedback! 🙏 🙏